pkg/services/sqlstore/dashboard_snapshot_test.go

```diff
@@ -15,6 +18,12 @@ func TestDashboardSnapshotDBAccess(t *testing.T) {
 	Convey("Testing DashboardSnapshot data access", t, func() {
 		InitTestDB(t)
 
+		origSecret := setting.SecretKey
```

Comment:
Related to other comment regarding GoConvey. These tests should be quite easy to convert to standard Go tests I think. However, this can be done as a separate follow-up PR if you want? 

Here's a good example of how we normally write standard tests using testify for assert/require: https://github.com/grafana/grafana/blob/master/pkg/infra/serverlock/serverlock_test.go

Some general testify 
- The `require` package you want to use if you want the test runner to fail directly (most common)
- The `assert` package` you can use if you have multiple asserts that doesn't depend on each other